recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
     number: 0
-    skip: True  # [py35]
+    skip: True  # [py3k]
```

Comment:
I have changed this to skip also py36 and future Python versions, this package is needed only for Python 2, 3.3 and 3.4.